deploys/demo-test.syncfor.science/docker-compose.yml

```diff
@@ -34,6 +34,15 @@ services:
      - VIRTUAL_PORT=5000
      - LETSENCRYPT_HOST=tests.demo-test.syncfor.science
      - LETSENCRYPT_EMAIL=letsencrypt@mg.syncfor.science
+  crucible:
+    networks:
+     - proxy-tier
+    environment:
+     - VIRTUAL_HOST=crucible.demo-test.syncfor.science
```

Comment:
https://crucible.demo-test.syncfor.science is working now (as well as anywhere else is).